.github/workflows/testing.sh

```diff
@@ -2,7 +2,7 @@
 
 set -eu -o pipefail
 
-export GRADLE_OPTS="-Xms128m"
+export GRADLE_OPTS="-Xms128m -Xmx512m"
```

Comment:
This now needs to be reverted, since the file is now deleted.

Reply:
I merged `master` and moved the change to `testing.yml`.

Reply:
Hmm... The commit messages for this change is strange message, as it doesn't match the change ta all: 17ffe1d4f9690311a814c85f90d333ea7de8afd0

I had assumed that the max heap size of 512 MB was necessary to fit within the runner's memory limits. But that message just confuses things.

Reply:
Yeah, re-reading the commit message I can see that it is misleading. The problem is that Gradle 6.9 adds `-Xmx64` as a Gradle option automatically. That means you can't use `-Xms128m`, since then the minimum heap would be bigger as the maximum heap which fails the java executable. Probably no Java options are necessary to run the build at all, let me try.

Reply:
I removed the `GRADLE_OPTS` and the build passes. Is everything good to go now?

Reply:
Ooh, I see. it did say _minimum_ was 64 MB. Interesting. That's small, but also makes it clear the type of memory usage that is acceptable ("If you're big, use another process!").

Yeah, I'm fine removing them instead. Obviously, things are different now with how Gradle uses memory.